javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,35 @@
+// Flight booking fullname function
+console.log("------------------ 1st Task ------------------");
+let fullname1 = getFullname("Elon", "Musk");
+let fullname2 = getFullname("Steve", "Jobs");
+
+console.log("1st Fullname: " + fullname1);
+console.log("2nd Fullname: " + fullname2);
+
+function getFullname(firstName, lastName, useFormalName = false){
+  if (useFormalName) {
+    return ("Lord " + firstName + " " + lastName);
+  } else {
+    return (firstName + " " + lastName);
+  }
+
+}
```

Comment:
You would usually place the function first, so it's "created" before you use it.